monitor/monitor.go

```diff
@@ -149,6 +170,38 @@ func New(opts MonitorOptions, logger *log.Logger, clk clock.Clock) (*Monitor, er
 			submitPreCert:      opts.SubmitOpts.SubmitPreCert,
 			submitCert:         opts.SubmitOpts.SubmitCert,
 		}
+		if db != nil {
+			keyHash := sha256.Sum256([]byte(opts.LogKey))
+			m.submitter.logID = big.NewInt(0).SetBytes(keyHash[:]).Int64()
```

Comment:
Why was this `logID` assignment necessary in this PR? Did I miss something?

Reply:
`certSubmitter` needs this to identify which log the cert was submitted to in the `AddCert` call.